index.md

```diff
@@ -0,0 +1,175 @@
+| vehicle | refer to [message VehicleDescriptor](#VehicleDescriptor) |
+|  | If separate `VehiclePosition` and `TripUpdate` feeds are provided, [TripDescriptor](#TripDescriptor) and [VehicleDescriptor](#VehicleDescriptor) ID values pairing should match between the two feeds.<br/>For example, a `VehiclePosition` entity has `vehicle_id:A` and `trip_id:4`, then the corresponding `TripUpdate` entity should also have `vehicle_id:A` and `trip_id:4`. If any `TripUpdate` entity has `trip_id:4` and any `vehicle_id` other than 4, this is an error. |
+| position |  |
+| current_stop_sequence |  |
+| stop_id | All `stop_ids`  must exist in the GTFS data in `stops.txt` and must have `location_type=0` in `stops.txt`. |
+| current_status |  |
+| timestamp | Even though this field is optional, providing the timestamp at which vehicle position was measured is strongly recommended. |
+| congestion_level |  |
+| occupancy_status |  |
+### Position
+
+| Field Name | Recommendation |
+| --- | --- |
+| latitude |  |
+| longitude | |
+| bearing |  |
+| odometer |  |
+| speed |  |
+### Alert
+
+| Field Name | Recommendation |
+| --- | --- |
+| active_period |  |
+| informed_entity | Specifiers (`agency_id`, `route_id`, etc.) in one [EntitySelector](https://gtfs.org/reference/realtime/v2/#message-entityselector) should be interpreted as being joined by the logical `AND` operator. Any combination of specifiers must exist in the GTFS data. |
+| cause |  |
+| effect |  |
+| url |  |
+| header_text |  |
+| description_text |  |
+| tts_header_text |  |
+| tts_description_text |  |
+| severity_level |  |
+### TripDescriptor
+
+| Field Name | Recommendation |
+| --- | --- |
+| trip_id | Developers need to make sure that all `trip_ids` must exist in the GTFS data, unless their `schedule_relationship` is set to `ADDED`. |
+|  | Frequency-based trips must contain `trip_id`, `start_time`, and `start_date`. |
+| route_id | Developers need to make sure that all `route_ids` must exist in the GTFS data. |
+| direction_id | Developers need to make sure that all `direction_id` must exist in the GTFS data. |
+| start_time | [Frequency-based trips](#Frequency-based-trips) must contain `trip_id`, `start_time`, and `start_date`. |
+| start_date | [Frequency-based trips](#Frequency-based-trips) must contain `trip_id`, `start_time`, and `start_date`. |
+| schedule_relationship | For [Frequency-based trips](#Frequency-based-trips), this should be `UNSCHEDULED` or empty. |
+|  | Trips that have `schedule_relationship:ADDED` must not be in GTFS data. |

```

Comment:
"must not" implies that this is an obligation and not a suggested Best Practice.